static/app/views/eventsV2/utils.tsx

```diff
@@ -55,12 +56,20 @@ const TEMPLATE_TABLE_COLUMN: TableColumn<React.ReactText> = {
 export function decodeColumnOrder(
   fields: Readonly<Field[]>
 ): TableColumn<React.ReactText>[] {
+  let equations = 0;
   return fields.map((f: Field) => {
     const column: TableColumn<React.ReactText> = {...TEMPLATE_TABLE_COLUMN};
 
     const col = explodeFieldString(f.field);
-    column.key = f.field;
-    column.name = f.field;
+    if (!isEquation(f.field)) {
+      column.key = f.field;
+      column.name = f.field;
+    } else {
+      const columnName = `equation[${equations}]`;
+      equations += 1;
+      column.key = columnName;
+      column.name = columnName;
+    }
```

Comment:
```suggestion
    const columnName = isEquation(f.field) ? `equation[${equations}]` : f.field;
    column.key = columnName;
    column.name = columnName;
```

Reply:
Wasn't able to do this suggestion cause I need to increment the equations counter, switched to a let here:
https://github.com/getsentry/sentry/pull/26413/commits/1d53fe589aa135e8542f0bf988405e7ed14699fe#diff-fd174f26fdf13d19c698526fa9e27dbd6d1d264093b228a6599f4eefccbdf49e